tests/benchmark/generate_csv.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+import csv
+import json
+import os
+import sys
+
+
+def generate_csv(filename, tag):
+    """Extract function name, time used and memory usage from the metadata and write to the output CSV file.
+
+    Args:
+        filename (str): Filename of the performance json file to read
+        tag (str): Tag of the docker container
+    """
+    with open(filename) as input:
+        data = json.load(input)
+    benchmarks = data["benchmarks"]
+    runtime_version = os.path.basename(filename).split(".json")[0]
+
+    # Write data to CSV file
+    with open("{}.csv".format(os.path.splitext(filename)[0]), "wb") as output:
+        csv_writer = csv.writer(output, delimiter=',')
+        for benchmark in benchmarks:
+            try:
+                # Get the function name
+                func_name = benchmark["metadata"]["name"]
+                # Get the time used for this function, convert to millisecond
+                time_used = benchmark["runs"][0]["values"][0] * 1000
```

Comment:
Can you add an explicit float() call here, to make it clear that it isn't supposed to be a string or list or something else?

    time_used = float(benchmark["runs"][0]["values"][0]) * 1000

Same with the memory usage.